frontend/views/containers/payments/PaymentRowReceived.vue

```diff
@@ -112,6 +118,9 @@ export default ({
         alert(e.message)
       }
     }
+  },
+  async mounted () {
+    this.relativeTo = await this.historicalPeriodStampGivenDate(this.payment.date)
```

Comment:
In the future, please place `mounted` near the top of the component, as it is some of the first code to be executed